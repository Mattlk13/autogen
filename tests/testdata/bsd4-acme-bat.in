-c "Acme Inc." -l bsd4 file.bat
