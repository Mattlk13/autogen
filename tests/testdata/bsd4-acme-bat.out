rem Copyright (c) 2014, Acme Inc.
rem All rights reserved.
rem
rem Redistribution and use in source and binary forms, with or without
rem modification, are permitted provided that the following conditions are met:
rem 1. Redistributions of source code must retain the above copyright
rem    notice, this list of conditions and the following disclaimer.
rem 2. Redistributions in binary form must reproduce the above copyright
rem    notice, this list of conditions and the following disclaimer in the
rem    documentation and/or other materials provided with the distribution.
rem 3. All advertising materials mentioning features or use of this software
rem    must display the following acknowledgement:
rem    This product includes software developed by Acme Inc..
rem 4. Neither the name of the Acme Inc. nor the
rem    names of its contributors may be used to endorse or promote products
rem    derived from this software without specific prior written permission.
rem
rem THIS SOFTWARE IS PROVIDED BY Acme Inc. ''AS IS'' AND ANY
rem EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
rem WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
rem DISCLAIMED. IN NO EVENT SHALL Acme Inc. BE LIABLE FOR ANY
rem DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
rem (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
rem LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
rem ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
rem (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
rem SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

rem TODO: High-level file comment.

